src/gui_common/tooltip/ToolTipCallbackData.cs

```diff
@@ -3,13 +3,16 @@
 /// <summary>
 ///   Helper class to contain the OnMouseEnter and OnMouseExit callback for the custom tooltips
 /// </summary>
-public class ToolTipCallbackData : Reference
+public class ToolTipCallbackData : Object
```

Comment:
Why was this base class change done?

Reply:
Was changed so a callback data instance could be freed manually on tooltip unregister, but that itself is not very safe which is why the call is set to run on next frame. Revisiting this, inheriting Reference here may actually be more convenient and safer as you could just rely on garbage collection so no need to manually freeing the instance after removing its reference from the `ToolTipCallbacks` list, a bonus perk is that it won't get deleted immediately afterwards.